google-api-go-generator/gen.go

```diff
@@ -874,6 +926,56 @@ func (a *API) GenerateCode() ([]byte, error) {
 	return clean, nil
 }
 
+func (a *API) insertSplitFileComment() {
+	if apisToSplit[a.Package()] {
+		a.pn("")
+		a.pn(splitFileSeperator)
+		a.pn("")
+	}
+}
+
+// splitFileHeading writes the file preamble used when generating a split file
+// client like compute.
+func splitFileHeading(w io.Writer, pkg string) {
+	pn := func(format string, args ...interface{}) {
+		_, err := fmt.Fprintf(w, format+"\n", args...)
+		if err != nil {
+			panic(err)
+		}
+	}
+
+	pn(`// Copyright %s Google LLC.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Code generated file. DO NOT EDIT.
+`, *copyrightYear)
```

Comment:
Nit: Any reason this is a multiline string and other examples are single line per print?
```suggestion
	pn("// Copyright %s Google LLC.", *copyrightYear)
	pn("// Use of this source code is governed by a BSD-style")
	pn("// license that can be found in the LICENSE file.")
	pn("")
	pn("// Code generated file. DO NOT EDIT.")
```

Don't mind either way.